### ISSUE TITLE
Setting default value for search box after loading options

### ISSUE THREAD

**Issue by landwire** [selectize/selectize.js#331]
I am loading options like so, which works fine:
Enhancement maybe:

Adding a parameter called "preselect" similar to "preload" and then you would choose a data-value from the loaded/available dropdowns. So you would call it like:
preselect: '135' or whatever the value is. That value would then show already in search box.

```
     $select_city = jQuery('#input_11_7').selectize({
        preload: true,
        load: function(query, callback) {
            select_city.clear();
            select_city.clearOptions();
            var value = 95;
            jQuery.each(lw_linked_selects.items, function() {
                var linked_item = this;
                //console.log(linked_item);
                if ( parseInt(value) > 0 ) {
                    select_city.enable();
                    jQuery.each(linked_item.terms, function() {
                        var term = this;
                        //console.log(term);
                        if (term.parent == value) {
                            results = [{ "termName"  : term.name, "termId" : term.id }];
                            console.log(results)
                            callback(results); 
                            //$target.append('<option value="'+term.id+'">'+term.name+'</option>');
                         }
                    });
                }
             });
        }, // city load end
        valueField  : 'termId',
        labelField  : 'termName',
        searchField : 'termName',
        create      : function(input) {
            jQuery("#input_11_5").val(input);
            //jQuery("#input_11_7").val(input); // not sticking on submission - why not???
            return {
                value: input,
                text: input
            }
        },
        sortField   : [{field: 'termName', direction: 'asc'}],
        maxItems    : 1,
        openOnFocus : false,
        hideSelected: true,
    //sortField: { "field"  : "labelField" },
    //onChange: function(value) {
               // new city value in here !
                //jQuery("#input_11_5").val(value);
      //      },
    });
```

How can I set one of the loaded options as a default value already showing inside the search box? Rather than have an empty search box.
Basically maybe adding a parameter called "preselect" similar to "preload" and then you would choose a data-value from the loaded/available dropdowns. So you would call it like:
preselect: '135' or whatever the value is.

**Comment by joallard** [comment]
Thanks for reporting this issue. I am closing it since the issue has been inactive for more than 2 years. If there is still a problem with the latest version, please re-open and if relevant, add details following the [Issue reporting guide](https://github.com/selectize/selectize.js/wiki/Reporting-issues-guide).
